src/main/kotlin/com/github/nowtilous/projectcolor/color_setters/LinuxColorSetter.kt

```diff
@@ -1,9 +1,16 @@
 package com.github.nowtilous.projectcolor.color_setters
 
-/**
- * Linux's layout is very similar to MacOs's, only difference
- * being the component tree path to the menu bar.
- */
-class LinuxColorSetter : MacOSColorSetter() {
-    override val TITLE_BAR_COMPONENT_PATH = listOf("JBLayeredPane", "JBPanel", "JBBox")
+import com.github.nowtilous.projectcolor.utils.getForegroundColorBasedOnBrightness
+import com.intellij.openapi.project.Project
+import java.awt.Color
+
+class LinuxColorSetter : ColorSetter() {
+    override val TITLE_BAR_COMPONENT_PATH = listOf("JBLayeredPane", "LinuxIdeMenuBar")
+
+    override fun setTitleBar(color: Color, project: Project) {
+        val titleBarComponent = findTitleBarComponent(project)
+
+        recursiveSetComponentColor(titleBarComponent, color, "background")
+        recursiveSetComponentColor(titleBarComponent, getForegroundColorBasedOnBrightness(color), "foreground")
+    }
```

Comment:
Testing in Ubuntu 22.04.1, that's the result:
![image](https://user-images.githubusercontent.com/20445645/202936454-39a2d63a-50ba-4d99-b6e9-b7627bf49637.png)

I could't color the whole extent of the menu bar yet, but I believe this is better than coloring the menu bar.